inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -262,18 +262,18 @@ class Core::Impl : public ICore {
     }
 
     CNNNetwork ReadNetwork(const std::string& modelPath, const std::string& binPath) const override {
-        IE_PROFILING_AUTO_SCOPE(Core::ReadNetwork)
+        OV_ITT_SCOPED_TASK(itt::domains::IE, "Core::Impl::ReadNetwork1");
```

Comment:
Are you trying to create separate library in this PR with keeping names as is or changing them in parallel? Anyway ReadNetowork1 and ReadNetwork2 look strange)

Reply:
I'll replace ReadNetwork1 with ReadNetwork but it will be very difficult to distinguish between the two functions with different signatures.

Reply:
@vladislav-volkov look at the `__PRETTY_FUNCTION__` macro. It seems it resolve problem with different signatures.

Reply:
I have added printout:
```
std::cout << __PRETTY_FUNCTION__ << std::endl;
```

Output:
```
virtual InferenceEngine::CNNNetwork InferenceEngine::Core::Impl::ReadNetwork(const string&, const string&) const
```

Reply:
Unfortunately __PRETTY_FUNCTION__ is compiler specific macro.
We should use more complicated macro like this https://www.boost.org/doc/libs/1_58_0/boost/current_function.hpp